web/beacon-app/src/features/home/components/GenerateApiKeyStep.tsx

```diff
@@ -21,7 +21,7 @@ export default function GenerateApiKeyStep() {
   const alreadyHasKeys = apiKeys?.api_keys?.length > 0;
 
   const onOpenGenerateAPIKeyModal = () => {
-    if (alreadyHasKeys) return;
+    // if (alreadyHasKeys) return;
```

Comment:
Should this be removed or will a separate story need to be created for this?

Reply:
good catch, I should bring it back.